src/installers.ts

```diff
@@ -527,22 +528,41 @@ export const wrapInstall =
     //
     // This is the function that Vortex calls
     async (
-      files: string[],
-      destinationPath: string,
+      filesRelativePaths: string[],
+      destinationDirPath: string,
       _gameId: string,
       _progressDelegate: VortexProgressDelegate,
     ): Promise<VortexInstallResult> => {
     //
       const vortexApi: VortexApi = { ...vortex.api, log: vortexApiThing.log };
 
       vortexApi.log(`info`, `Trying to install using ${installer.type}`);
-      vortexApi.log(`debug`, `Input files:`, files);
+      vortexApi.log(`debug`, `Input files:`, filesRelativePaths);
+
+
+      const treeForInstallers =
+        unwrapTreeIfNecessary(vortexApi, fileTreeFromPaths(filesRelativePaths));
 
-      const treeForInstallers = unwrapTreeIfNecessary(vortexApi, fileTreeFromPaths(files));
       const sourceFileCount = fileCount(treeForInstallers.fileTree);
 
-      const modInfo = modInfoFromArchiveNameOrSynthetic(destinationPath);
-      vortexApi.log(`info`, `Extracted mod info from path: ${destinationPath}`);
+      const archivePathOnDisk = destinationDirPath;
+
+      const archivePath =
+        treeForInstallers.transform !== Transform.Unwrapped
+          ? archivePathOnDisk
+          : pipe(
+            archivePathOnDisk.split(path.sep),
+            filter(not(pathEq(treeForInstallers.wrapperDir))),
+            (parts) => path.join(...parts),
+          );
```

Comment:
I wonder if something similar could be done for path separator normalization across the board so that tests could be run independent of OS (linux wants forward slashes, windows like backwards slashes, etc).

Reply:
Yep, that’s what `path{Eq,In,Contains}` are for! I just didn’t have time/energy to go through all the places that need to be changed.